code/addons/a11y/src/a11yRunner.ts

```diff
@@ -1,22 +1,25 @@
+import { deprecate } from 'storybook/internal/client-logger';
+import { ElementA11yParameterError } from 'storybook/internal/preview-errors';
+
 import { global } from '@storybook/global';
 
-import type { AxeResults } from 'axe-core';
+import type { AxeResults, ContextProp, ContextSpec, Selector, SelectorList } from 'axe-core';
 import { addons } from 'storybook/preview-api';
 
 import { EVENTS } from './constants';
-import type { A11yParameters } from './params';
+import type { A11yParameters, SelectorWithoutNode } from './params';
 
 const { document } = global;
 
 const channel = addons.getChannel();
 
-const defaultParameters = { config: {}, options: {} };
+const DEFAULT_PARAMETERS = { config: {}, options: {} } as const;
```

Comment:
Not sure why these should be all caps? I usually only use all caps for primitive values, and they should be in a constants file. Typically such things are then used in multiple places. These don't seem to be reusable constants like that, they're just local variables.

Reply:
I see what you mean. My heuristic is to use SNAKE_CASE when something is a _constant_ that is not supposed to be changed (regardless of it being a primitive or not), and only move it to a `constants` file if it is actually shared across multiple files.

But no strong opinion.